### DISCUSSION TITLE
App feature suggestion

### DISCUSSION THREAD

**Discussion by LordFlaya** [community/community#124823] [Mobile]
Please add the ability to select and share repos.

**Comment by kangarruu** [comment]
Hi @LordFlaya, thanks for writing in! Would you elaborate a bit on where you'd like to see that functionality? If you visit a particular Repository's profile page you can share it from the menu bar. Do you mean selecting multiple repositories to share at one time? Would you want to share from your Favorites, or from a feed like the Explore tab? I'll open up a feature request issue so that we could keep track of your feedback. Thanks!

**Reply by LordFlaya** [reply]
I see that I was a bit vague in my feedback.

Yes, I would like to be able to select multiple repos from i.e favorites or search results, so I guess basically a universal share option across the app.

**Reply by kangarruu** [reply]
That does sound handy 😃 I'll open up an issue and we'll keep it in mind for future work! Thanks again for sharing your feedback.
